datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearch.tsx

```diff
@@ -188,6 +188,7 @@ export const EmbeddedListSearch = ({
         variables: {
             input: searchInput,
         },
+        fetchPolicy: 'cache-first',
```

Comment:
oh man what a time to be alive - these possibleTypes making search caching possible!